src/test/java/persistence/sql/ddl/ColumnMetaInfoFactoryTest.java

```diff
@@ -0,0 +1,43 @@
+package persistence.sql.ddl;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.entity.Person;
+
+import java.lang.reflect.Field;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class ColumnMetaInfoFactoryTest {
+
+    @Test
+    @DisplayName("지원 하는 Annotation 테스트")
+    public void supportedAnnotationTest() throws NoSuchFieldException {
+        Field field = Person.class.getDeclaredField("id");
+
+        List<ColumnMetaInfo> columnMetaInfos = ColumnMetaInfoFactory.createColumnMetaInfo(field);
+
+        assertThat(columnMetaInfos.get(0).getValue()).isEqualTo("AUTO_INCREMENT");
```

Comment:
```
columnMetaInfos.get(0)
columnMetaInfos.get(1)
```

이러한 코드 패턴은 0번째와 1번째 위치에 어떤 값이 들어가야하는지에 대한 제약 사항을 명확하게 전달하지 못할 가능성이 커요.
이로 인해 코드의 가독성과 유지 보수가 어려워질 수 있어요. 
이 문제를 해결하기 위해 [일급 컬렉션 (First Class Collection)](https://jojoldu.tistory.com/412)을 사용하는 것을 고려해보는 것은 어떨까요?

Reply:
```suggestion
       ColumnMetas columnMetas = ColumnMetaInfoFactory.createColumnMetaInfo(field);

        assertThat(columnMetas.identityColumnString()).isEqualTo("AUTO_INCREMENT");
```

`List` 자료구조 보다는, 우리만의 컬럼 메타 정보를 가진 `ColumnMetas`(일급콜렉션)을 만든다면 그러한 객체에게 질문을 던저 볼 수 있을것 같아요.

이렇게요,
`식별자 컬럼 전략이 뭐야?` =>  `columnMetas.identityColumnString()`

**### 저의 관점일 뿐이며, 자유롭게 고민해주세요~**
>(구현하신 ColumnMetas의 활용 개념과 저의 관점이 다를수 있지만 이런관점도 있을수 있다 정도로 봐주세요!)